spec/integration/preview_spec.rb

```diff
@@ -32,7 +50,17 @@
         let (:template) { id }
         let (:fixture) { id }
 
-        specify { expect(response).to eq("content") }
+        specify { expect(response).to eq("{\"content\":\"as json\"}") }
+
+        expected_headers = {
+          "Content-Type"                => "application/json",
+          "Access-Control-Allow-Origin" => "*",
+          "X-Sequence"                  => expected_time,
+          "Content-Length"              => "21",
+          "X-Content-Type-Options"      => "nosniff"
+        }
+
+        specify { expect(last_response.headers).to eq(expected_headers) }
```

Comment:
Me personally, i'm not a massive fan of the `specify` style of testing in rspec - I think it's much better to have a descriptive `it` block...

i.e. it "will respond with the correct HTTP headers"